src/theme/foundations/breakpoints.ts

```diff
@@ -1,22 +1,21 @@
 /**
- * Breakpoints from "src/styles/base/_01-breakpoints.scss"
  * How to use Chakra responsive styles: https://chakra-ui.com/docs/features/responsive-styles
  *
- * Chakra Value |     DS Variable           | EM/PX value
- * ------------------------------------------------------
- *      sm      | --nypl-breakpoint-small   | 20em/320px
- *      md      | --nypl-breakpoint-medium  | 37.5em/600px
- *      lg      | --nypl-breakpoint-large   | 60em/960px
- *      xl      | --nypl-breakpoint-xl      | 80em/1280px
- *     2xl      |        N/A                | 96em/1536px
+ * Chakra Value |     DS Variable           | EM/PX value    | NYPL reference
+ * --------------------------------------------------------------------------
+ *      sm      | --nypl-breakpoint-small   | 30em / 480px   | small mobile
```

Comment:
In `/StyleGuide/Breakpoints.mdx` and here, because the names don't map 1-to-1 from the semantic meaning to the CSS variable, it makes it very confusing. I guess semantically we want to think about 768px as "large mobile" but programmatically this is `--nypl-breakpoint-mdeium` and `isLargerThanMobile` in the hook. It may not be a big deal but I wonder if it will be confusing for devs.

Reply:
The t-shirt sizes comes from the Chakra syntax, but we can certainly expand on that and have duplicate CSS vars that map to the same breakpoint values: one aligning with Chakra and one aligning with the new semantic naming convension.

Reply:
Yes, and then it's up to the dev to use the one that makes sense for them.

Reply:
Ok, I will add some new values.